client/src/pages/Messages/Conversation/Conversation.tsx

```diff
@@ -21,28 +76,25 @@ export default function Conversation(): JSX.Element {
                   }}
                   classes={{ badge: classes.inactiveBadge }}
                 >
-                  <Avatar alt="profile picture" src={profilePic} />
+                  <Avatar alt="profile picture" src={profilePicAvatar} />
                 </Badge>
               </ListItemAvatar>
             </Grid>
             <Grid item xs={10}>
               <ListItemText disableTypography style={{ paddingLeft: '0.5em' }}>
                 <Typography variant="h6" style={{ fontWeight: 700 }}>
-                  John Doe
+                  {participant?.username}
                 </Typography>
                 <Typography variant="body1" className={classes.messageSnippet}>
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
-                  dolore magna aliqua.
+                  {lastMessage ? lastMessage.message : null}
```

Comment:
When we have no last message, maybe show some meaningful text (no message history) rather than null :)

Reply:
Didn't put a lot of thought into it. I will replace "null" with "No Message History".